wagtail/contrib/simple_translation/templates/simple_translation/admin/input_option.html

```diff
@@ -1,6 +1,6 @@
 {% if widget.wrap_label %}
     <label{% if widget.attrs.id %} for="{{ widget.attrs.id }}"{% endif %}{% if widget.attrs.disabled %} class="disabled"{% endif %}>
-        {% include "django/forms/widgets/input.html" %} {{ widget.label }}
+        {% include "django/forms/widgets/input.html" %}{{ widget.label }}
```

Comment:
this space may be intentional, best to avoid changes like this unless there is an explicit reason. I can fix this up.